### ISSUE TITLE
Pages added twice to archive after migrating from git to Dropbox

### ISSUE THREAD

**Issue by PetarPeychev** [davidmerfield/Blot#310]
After moving my blog from git to Dropbox, the pre-existing files were not added to the Dropbox folder. I uploaded them manually to it and now the the archive contains two links to each post both pointing to the same page.

![screen](https://user-images.githubusercontent.com/33529197/87256428-6ce99600-c48a-11ea-83b0-9e8c978cfa90.png)

Removing / adding an underscore to the .md files in Dropbox removes the duplicate links, but the pages are also removed and the links point to nothing.

**Comment by PetarPeychev** [comment]
I just noticed the main feed also has two copies of each article in the same order as in the archive.

**Comment by davidmerfield** [comment]
Hello Petar – sorry for the delay – please consider emailing [support@blot.im](mailto:support@blot.im) for a more prompt response in future, I check email much more regularly than GitHub issues. I noticed that you reset your blog – is everything in order now?

---

As for the bug itself – this happens because Git is case-sensitive and Dropbox is not. The Dropbox client stores all its files in the lowercase. When transferring from Git to Dropbox. I will fix this and follow up with you

**Comment by PetarPeychev** [comment]
Yeah, I did reset and fix it manually. Thanks for you support of the project, I will try to use the email if there are any more problems in the future.

**Comment by davidmerfield** [comment]
I looked into this a little further and realized the problem is that there isn't a blocking 'transfer' stage when you connect the Dropbox client to an existing folder – there needs to be a progress bar which indicates as the files are transferring. 

> After moving my blog from git to Dropbox, the pre-existing files were not added to the Dropbox folder.

Did you encounter any errors when you were changing clients? Did a page load time out at any point?

**Comment by PetarPeychev** [comment]
I don't recall anything like that, but I might not be remembering it.

**Comment by eg1776** [comment]
I also ran into this issue.

**Comment by davidmerfield** [comment]
@ericgregorich Same question to you: did you encounter any errors when you were changing clients? Did a page load time out at any point?

**Comment by ericgregorich** [comment]
> @ericgregorich Same question to you: did you encounter any errors when you were changing clients? Did a page load time out at any point?

No errors. I just had written a single "Test" post. After switching from Git to Dropbox I then had a duplicate of that post. I only saw one in my files and on my account page. I deleted it but I still had one Test file remaining. I couldn't figure out where it was coming from so I just deleted the site and started over.

**Comment by davidmerfield** [comment]
@ericgregorich Noted, thank you for the information

**Comment by davidmerfield** [comment]
Minimal steps to reproduce in production:

1. Set up empty blog with git client
2. Add file whose name contains uppercase character, e.g. *Article.txt* 
3. Switch to Dropbox client
4. You now how two posts from Article.txt

**Comment by davidmerfield** [comment]
This was fixed in #464 

Thank you all for the reports

**Comment by dvcrn** [comment]
I ran into the same thing (in December 2021). A export of the entire site data shows that the same post exists twice, with identical guid. 

Even after I removed all posts from the blog it still displays the duplicates, but is not able to navigate to their detail page. The only way to resolve this was to wipe everything and start from scratch

Since this is quite recent it looks like https://github.com/davidmerfield/Blot/pull/464 did not fix the issue
